tests/tests/optionsTests.robot

```diff
@@ -124,4 +125,96 @@ Verify Output log to a file option
     Click Load Button
     Wait Until Keyword Succeeds  5s  1s  Click Element  ${DownloadLogsButton}
     Switch Window    new
-    Wait Until Element Is Visible    ${OpeningFileText}
\ No newline at end of file
+    Wait Until Element Is Visible    ${OpeningFileText}
+
+Verify Extraction filter set to 'Chosen language and language agnostic files'
+    [Documentation]    Chosen language and language agnostic files extraction filter option
+    [Tags]    option    extraction filter
+
+    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${extraction_filter}[archive_name].zip
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${OptionsList}
+    Click Add Files Button
+    Upload Test File    ${extraction_filter}[path]
+    Select From List By Index    ${ExtractionLangFilterOption}    0
+    List Selection Should Be     ${ExtractionLangFilterOption}    lang-plus-agn
+    Select From List By Index    ${CollisionResolutionOption}    1
+    List Selection Should Be     ${CollisionResolutionOption}    rename
```

Comment:
Does Collision Resolution has to be set to "rename" or can be default "overwrite"?

Reply:
It needs to be set to overwrite in case if we have duplicated file name in archive. E.g like in our test we have 3 Readme.txt files each for different language. If we leave Collision resolution to "overwrite" we get only one Readme file as output instead of expected 3.